### ISSUE TITLE
A11y_PowerShellEditorServices_Terminal_ColorContrast : Color Contrast ratio is less than 4.5:1 when we apply color themes.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61704]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10

**Additional Details:**
Color contrast MAS1.4.3

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "View" item of menu bar and select "Integrated Terminal"->Navigate to "Terminal" tab and select "PowerShell Integrated Console" from combo box.
3.Navigate "Terminal multiline editable box & write any query & press enter 
4. You will get error message 
3.Navigate to Setting button of Activity Bar and select "ColorTheme" option.
4.Color Themes shows number of different colors list apply all that theme on the editor.
5.Open colorcontrast analyser and check the colorcontrast ratio.
6.It shows colorcontrast ratio less than expected colorcontrast ratio.
 
**Actual:**
After applying following color themes Color contrast ratio is less than 4.5:1.
1)Powershell ISE theme
2)Solarized light theme
3)Dark (Visual Studio) theme
4)Kimbie Dark theme
5)Monokai theme
6)Solarized Dark theme

**Expected:**
The expected colorcontrast ratio should be greater than 4.5:1.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
User will not able to see the error content on the terminal.

**MAS Reference:**
MAS1.4.3- https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D

Does this issue occur when all extensions are disabled?: Yes
![a11y_powershelleditorservices_terminal_colorcontrast](https://user-images.githubusercontent.com/39204909/47426776-cf2ccf80-d7ab-11e8-9ebd-aae5258f34dd.png)
![a11y_powershelleditorservices_terminal_colorcontrast1](https://user-images.githubusercontent.com/39204909/47426786-d5bb4700-d7ab-11e8-83e5-b3d29f9af6ad.png)
